timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -172,44 +168,37 @@ def _create_mock_event(event_id, quantity, time_diffs=None,
         time_diffs.extend([time_diffs[len(time_diffs) - 1]] *
                           (quantity - len(time_diffs)))
 
-    # Setup for Event object initialisation
-    ds = MockDataStore('test', 0)
-    user = User('test_user')
-    sketch = Sketch('test_sketch', 'description', user)
-    label = sketch.Label(label='Test label', user=user)
-    sketch.labels.append(label)
-
     event_timestamp = 1410895419859714
-    event_template = ds.get_event('test', 'test')
```

Comment:
Was used to extract the event_dict template, stored in the MockDataStore, and then passed to _create_eventObj, but since it can be done more in place in the _create_eventObj function, it was removed from here.